docs/components/modules/ROOT/pages/properties-component.adoc

```diff
@@ -368,7 +368,7 @@ The default mode is that both of them are in override mode, and they are check i
 The check stops at first found property value for the key.
 
 You can control these modes using the `systemPropertiesMode` and `environmentVariableMode`
-options on the properties component.
+options on the property's component.
```

Comment:
IMO the original wording is correct given that the name of the component is `properties`.

Reply:
sure thanks.